powerauth-java-cmd-lib/src/main/java/io/getlime/security/powerauth/lib/cmd/steps/v2/PrepareActivationStep.java

```diff
@@ -85,6 +85,7 @@ public JSONObject execute(StepLogger stepLogger, Map<String, Object> context) th
 
         if (stepLogger != null) {
             stepLogger.writeItem(
+                    "activation-prepare-start",
```

Comment:
I would use `create-activation` prefix here. See my related comment in `CreateActivationStep`.